app/src/main/res/values/array.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="verification_methods">
+        <item>Visual verification</item>
+        <item>Rhythm verification</item>
+        <item>Choose when pairing</item>
+    </string-array>
```

Comment:
@ericcornelissen I think it's need to have this file if we would like to expand it with more static arrays. The name 'array.xml' is the default name for arrays, what would you give as a different name?